scripts/docgen/generate-docs.js

```diff
@@ -240,6 +243,63 @@ function fixAllLinks(htmlFiles) {
   return Promise.all(writePromises);
 }
 
+/**
+ * Generate an temporary abridged version of index.d.ts used to create
+ * Node docs.
+ */
+async function generateNodeSource() {
```

Comment:
Should this function be moved to generate-node-source.js ?

Reply:
That file was a temp file and shouldn't have been included. This function currently shares some global variables so it's harder to break out.  I could make another file for helper functions at some point but they all pass around global variables right now.